tasks/move-granules/tests/test-index.js

```diff
@@ -36,11 +37,16 @@ async function uploadFiles(files, bucket) {
 }
 
 async function updateFileTags(files, bucket, TagSet) {
-  await Promise.all(files.map((file) => s3().putObjectTagging({
-    Bucket: bucket,
-    Key: parseS3Uri(file).Key,
-    Tagging: { TagSet }
-  }).promise()));
+  await Promise.all(files.map((file) => s3PutObjectTagging(
+    bucket,
+    parseS3Uri(file).Key,
+    { TagSet }
+  )));
+}
+
+function granulesToFileURIs(granules) {
+  const m = granules.reduce((arr, g) => arr.concat(g.files.map((file) => file.filename)), []);
```

Comment:
Can we have a more specific variable name than `m`?

Reply:
72ce92679